### ISSUE TITLE
[package] jasper/2.0.33: Build fails on GitHub Actions due to misconfigured validate check

### ISSUE THREAD

**Issue by AnotherFoxGuy** [conan-io/conan-center-index#14682]
### Description

jasper used to build without issues on GitHub Actions until this line got added:
https://github.com/conan-io/conan-center-index/blob/master/recipes/jasper/all/conanfile.py#L64-L65

This killed all our windows builds on GitHub Actions, so would it be possible to check if it is actually running on CCI?

### Package and Environment Details

* Package Name/Version: **jasper/2.0.33**

See for further details: https://github.com/actions/runner-images/blob/win19/20221204.2/images/win/Windows2019-Readme.md

### Conan profile
```
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=Visual Studio
compiler.runtime=MD
compiler.version=16
os=Windows
os_build=Windows
[options]
[conf]
[build_requires]
[env]
```

### Steps to reproduce

Create a GitHub Action that runs this command: `conan.exe install jasper/2.0.33 --build missing --settings arch=x86_64 --settings build_type=Release --settings compiler=Visual Studio --settings compiler.version=16 --settings compiler.runtime=MD` 

### Logs

```
ERROR: There are invalid packages (packages that cannot exist for this configuration):
jasper/2.0.33: Invalid ID: jasper Current can not build in CCI due to windows SDK version. See https://github.com/conan-io/conan-center-index/pull/13285 for the solution hopefully
```
You can find the full log here: https://github.com/RigsOfRods/rigs-of-rods/actions/runs/3667066835/jobs/6199281985

**Comment by markaren** [comment]
Had the same issue on multiple Windows computers in conjunction with OpenCV.

**Comment by HappySeaFox** [comment]
Please vote for https://github.com/conan-io/conan-center-index/issues/13159 . This is the reason why jasper has this `raise` line.

**Comment by saphiresouldier** [comment]
Having the same issue on a windows computer!

**Comment by jcar87** [comment]
Hi! While we work on deploying a proper fix for #13159  - I believe this is already resolved in https://github.com/conan-io/conan-center-index/pull/13285 which was merged earlier today.

I'll be closing this issue as resolved since that check is now removed, but please reopen this issue if anything is missing!

Thanks @AnotherFoxGuy, @markaren , @HappySeaFox, @saphiresouldier
